.github/workflows/build-and-test.yml

```diff
@@ -208,6 +208,10 @@ jobs:
             ~/go/bin
             ~/go/pkg/mod
           key: go-cache-${{ runner.os }}-${{ hashFiles('**/go.sum') }}
+      - name: Setup Oracle DB Docker Image Dependency
+        shell: bash
+        run: |
+          docker pull container-registry.oracle.com/database/express:latest
```

Comment:
I don't think you need to explicitly pull the image. Integration tests should handle this